### PR TITLE
[Converged-MME]ts9_timer edit

### PR DESCRIPTION
**Pull request by Vyas20**
[lte][converged-mme][agw]

## Summary
Part of replacing the existing timers with zmq timers, this branch is forked from the branch in PR #4786  and contains ts9_timer edits.

## Test Plan
Testing with S1-AP Tester and dsTester.

### CODE REVIEW

**Comment by Vyas20** [comment]
MAGMA-OAI-TESTS

**Comment by Vyas20** [comment]
MAGMA-OAI-TESTS

**Comment by rdefosse** [comment]
MAGMA-OAI-TESTS

**Review by ulaskozat — CHANGES_REQUESTED** [review]
Still have the file MmeUeContext.cpp/h

File: lte/gateway/c/oai/tasks/mme_app/mme_app_sgs_detach.c [hunk 1]
```diff
@@ -380,27 +357,33 @@ void mme_app_send_sgs_imsi_detach_indication(
...
         LOG_MME_APP,
         "Invalid UE context received, MME UE S1AP Id: " MME_UE_S1AP_ID_FMT "\n",
         mme_ue_s1ap_id);
-    OAILOG_FUNC_OUT(LOG_MME_APP);
+    OAILOG_FUNC_RETURN(LOG_MME_APP, RETURNerror);
   }
   if (ue_context_p->sgs_context == NULL) {
     OAILOG_ERROR(
         LOG_MME_APP,
         "SGS EPS Detach Timer expired but no assoicated SGS context for UE "
```
Comment (ulaskozat) [inline lte/gateway/c/oai/tasks/mme_app/mme_app_sgs_detach.c]: fix the typo `assoicated`

**Comment by Vyas20** [comment]
MAGMA-OAI-TESTS

File: lte/gateway/c/oai/tasks/mme_app/mme_app_sgs_detach.c (hunk 1 above)
Comment (Vyas20) [reply lte/gateway/c/oai/tasks/mme_app/mme_app_sgs_detach.c]: Resolved

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**